scripts/lib/setup_venv.py

```diff
@@ -99,6 +102,13 @@
     "gifsicle",
 ]
 
+def get_venv_dependencies(vendor, os_version):
+    # type: (str, str) -> List[str]
+    if vendor == 'ubuntu' and os_version == '20.04':
+        return VENV_DEPENDENCIES + [PYTHON_DEV_DEPENDENCY.format("2"), ]
+    else:
+        return VENV_DEPENDENCIES + [PYTHON_DEV_DEPENDENCY.format(""), ]
```

Comment:
If we're doing this refactor to make this a function, should we include support for `yum` platforms in here as well?

Reply:
@timabbott I think its more of temporary solution as we'll remove the need of PYTHON_DEV_DEPENDENCY when upgrading Thumbor but if you want to return yum dependencies through this function we could proceed in that direction. Also, do you mean to include THUMBOR_VENV_DEPENDENCIES too?

Reply:
No, I'm just thinking that we could have this function return the correct dependencies for RHEL and Fedora rather than having `provision` import separate `COMMON_YUM_DEPENDENCIES`, for a cleaner interface with `provision`.  This can be a follow-up commit.